typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -214,6 +219,18 @@ export class HyperlaneIsmFactory extends HyperlaneApp<FactoryFactories> {
     return IAggregationIsm__factory.connect(address, signer);
   }
 
+  private async deployOpStackIsm(chain: ChainName, config: OpStackIsmConfig) {
+    const signer = this.multiProvider.getSigner(chain);
+    const overrides = this.multiProvider.getTransactionOverrides(chain);
+
+    const deployedIsm = await this.multiProvider.handleDeploy(
+      chain,
+      new OPStackIsm__factory(signer),
+      [config.nativeBridge, overrides],
+    );
+    return deployedIsm;
```

Comment:
signer and overrides stuff is done in `handleDeploy`
```suggestion
    return this.multiProvider.handleDeploy(
      chain,
      new OPStackIsm__factory(),
      [config.nativeBridge],
    );
```